graphql/queries/user.ts

```diff
@@ -48,3 +48,21 @@ export const GET_PROFILE = gql`
     }
   }
 `;
+export const GET_TRAINEE_PROFILE = gql`
```

Comment:
Instead of creating another query you can add resume in the `GET_PROFILE` query and use the only one query to fetch profile